status-im/status-react
# Add centralized metrics

This commit adds support for pushing centralized metrics to mixpanel.
    
It uses an interceptor and only picks a few selected events to push
through.
    
In order to test:
    
1) Create an account in the app
2) Go to Settings->Privacy
3) Enable metrics
    
You should now see the events on the mixpanel dashboard (login with your status-im account for access).

Only some example events are tracked, they are just for testing

It also address a couple of minor issue:
1) Backup seed phrase was shown as active when syncing, while it wans't there (just empty string)
2) Mark all read wasn't available on spectated communities
